requirements.txt

```diff
@@ -1,3 +1,3 @@
 requests==2.19.1
-siggen==0.1.2
+siggen<2
```

Comment:
If we restrict it to `<2`, then it'll pick up the latest version that's below major version 2. I'll keep to semantic versioning, so that should work and won't break the API.

Reply:
Wouldn't it be safer and more predictable to specify a single version?  Is the reasoning for this so that signature lists can be kept up to date?

Reply:
I just read #4 so this makes sense to me.

Reply:
This is one of those tough decisions: what to pin on. I was thinking that we control siggen, so we can make sure that anything under 2.0.0 guarantees API backwards compatibility. I think with that guarantee, it's safe to do this here and doing this here means we don't have to do an fx-crash-sig update for every siggen update. I think we want to have to do fewer fx-crash-sig updates.

Having said that, I don't really understand the Telemetry code or databricks or how this would get used. Is it likely that one of those systems will cache fx-crash-sig? In which case if we don't increment the fx-crash-sig version, they won't pick up newer versions of siggen, either?

Reply:
If I remember correctly databricks just does a pip install for each new cluster without caching so I think in this case it would install the latest siggen even if fx-crash-sig isn't updated.  You may want to double check that with someone else.